packages/amplify-category-geo/src/service-utils/mapParams.ts

```diff
@@ -70,6 +74,12 @@ export const getGeoMapStyle = (dataProvider: DataProvider, mapStyleType: MapStyl
     if (dataProvider === DataProvider.Here) {
       return `VectorHere${mapStyleType}`;
     }
+    else if (dataProvider === DataProvider.Here && mapStyleType === HereMapStyleType.RasterSatellite) {
+      return MapStyle.RasterHereExploreSatellite;
+    }
+    else if (dataProvider === DataProvider.Here && mapStyleType === HereMapStyleType.HybridSatellite) {
+      return MapStyle.HybridHereExploreSatellite;
+    }
```

Comment:
Something within this conditional is failing because the else ifs are going to always be false according to the unit tests.

Reply:
specifically, they say : 
```This condition will always return 'false' since the types 'DataProvider.Esri' and 'DataProvider.Here' have no overlap```

Reply:
Fixed.